__PROJECT_NAME__/.travis.yml

```diff
@@ -0,0 +1,34 @@
+language: objective-c
```

Comment:
I'm wary that this is considerably more complex than the previous travis file, understanding it requires a lot more knowledge about the shell and Travis CI. The majority of CocoaPods users probably don't know this stuff, can it be simplified?